python/pyspark/tests/test_context.py

```diff
@@ -213,6 +213,10 @@ def test_progress_api(self):
             rdd = sc.parallelize(range(10)).map(lambda x: time.sleep(100))
 
             def run():
+                # When thread is pinned, job group should be set for each thread for now.
+                # Local properties seem not being inherited like Scala side does.
+                if os.environ.get("PYSPARK_PIN_THREAD", "false").lower() != "true":
```

Comment:
Is it a typo for `true`? As it claims `When thread is pinned, job group should be set.` Or I misread the comment.

Reply:
Damn .. yes, it's a typo.. I was testing all PySpark tests when enabling this and forgot to bring this back .. thanks for catching this.